### PR TITLE
[FIX] website_blog: resolve scroll to content traceback on blog page

### PR DESCRIPTION
**Pull request by paru-odoo**
Resolve a traceback caused by the removal of the duration parameter during the refactoring of legacy code in dom.js. While, calling function are passing duration as params which remove in this PR.

Similarly, extract all the unused code from scrollTo functionality.

task-4095403

### CODE REVIEW

File: addons/web_editor/static/src/js/editor/snippets.options.js [hunk 1]
```diff
@@ -5927,10 +5927,8 @@ registry.SnippetMove = SnippetOptionWidget.extend(ColumnLayoutMixin, {
             const hidden = elTop < 0 || bottomHidden;
             if (hidden) {
                 scrollTo(this.$target[0], {
-                    extraOffset: 50,
-                    forcedOffset: bottomHidden ? heightDiff - 50 : undefined,
```
Comment (divy-odoo) [inline addons/web_editor/static/src/js/editor/snippets.options.js:5931]: Why we ignored the forcedOffset here? It recently added by the [PR]( https://github.com/odoo/odoo/pull/108831/files) so removing it should be breaking something.

Comment (divy-odoo) [reply addons/web_editor/static/src/js/editor/snippets.options.js:5931]: https://www.awesomescreenshot.com/video/31366815?key=3c389504c1efa4643df296d8c975b00b

May be this is what its breaking. :smiling_face_with_tear:

File: addons/web_editor/static/src/js/editor/snippets.options.js [hunk 2]
```diff
@@ -8859,45 +8857,6 @@ registry.BackgroundPosition = SnippetOptionWidget.extend({
         this.$target.css('background-position', '');
         this.$target.css('background-size', widgetValue !== 'repeat-pattern' ? '' : '100px');
     },
-    /**
-     * Saves current background position and enables overlay.
-     *
-     * @see this.selectClass for params
-     */
-    backgroundPositionOverlay: async function (previewMode, widgetValue, params) {
```
[the remark this answers is not in the document]
Comment (sobo-odoo) [reply addons/web_editor/static/src/js/editor/snippets.options.js:8867]: I confirm it is not dead code, it is used by this option, to adjust the background position :wink: 
image

**Comment by sobo-odoo** [comment]
Closing this, as the issue was fixed by [this PR](https://github.com/odoo/odoo/pull/179569) :slightly_smiling_face:

**Pull request closed without merging**